eval/eval.go

```diff
@@ -36,13 +36,15 @@ const (
 const (
 	defaultValuePrefix = "▶ "
 	initIndent         = vals.NoPretty
+	defaultSuppressBackground = "false"
```

Comment:
This constant belongs together with `defaultValuePrefix`, so move it up one line.